charts/prometheus-postgres-exporter/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: "0.11.1"
 description: A Helm chart for prometheus postgres-exporter
 name: prometheus-postgres-exporter
-version: 4.4.4
+version: 4.4.5
```

```suggestion
version: 4.5.0
```

I would suggest increasing the minor version number, a new feature is being introduced after all.

---

true